config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+
+
+
+
+
+
+  get 'outings' => 'outings#index'
```

Comment:
@Dpalazzari Any special reason to use get here instead of `resources :outings, only: [:index]`? Discussing with @njgheorghita and @drod1000 we feel we should go with the convention.